frontend/pages/hosts/details/cards/Activity/ActivityItems/InstalledSoftwareActivityItem/InstalledSoftwareActivityItem.tsx

```diff
@@ -24,15 +25,21 @@ const InstalledSoftwareActivityItem = ({
     <b>{actorName ?? "Fleet"}</b>
   );
 
+  let installedSoftwarePrefix = getInstallStatusPredicate(status);
+  if (tab !== "past" && activity.fleet_initiated) {
+    installedSoftwarePrefix =
+      status === "pending_uninstall" ? "will uninstall" : "will install";
+  }
```

Comment:
There's probably a cleaner way to do this but I'm really not familiar with typescript, thoughts/suggestions welcome. The `getInstallStatusPredicate` helper is still useful because it is used in global activities too, but not as much since we need to patch the prefix here anyway...